modules/mobile/spec/request/authorized_services_spec.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require_relative '../support/helpers/iam_session_helper'
+require_relative '../support/matchers/json_schema_matcher'
```

Comment:
Doesn't look like we're actually using the json schema matchers. Maybe we should be. Frankly, we've probably done a very poor job of keeping up on those and idk how much we care. 

Also not sure if we're actually using the common/client/errors below either.